scripts/zones/Abyssea-Grauberg/npcs/qm_xibalba.lua

```diff
@@ -1,15 +1,15 @@
 -----------------------------------
 -- Zone: Abyssea-Grauberg
---  NPC: qm_xiabalba (???)
--- Spawns Xiabalba
+-- NPC: qm_xibalba (???)
```

Comment:
you don't have to change this back, but just fyi for the future you can ignore the spacing of these till we set a pattern in stone in the style guide. so that we don't have multiple editors changing these header sections back and forth in multiple competing styles.